sdks/Cordova.md

```diff
@@ -0,0 +1,136 @@
+# Apache Cordova AeroGear Services
+
+## Motivation
+
+Provide set of the SDK's that can be used for Cordova based development.
+Provide best performance and user experience on the platform.
+
+## Proposed solution 1
+
+Pure javascript solution for the mobile configuration and orchestration 
+that will allow cooperability between JavaScript and Native.
+Configuration parser and other pure JS code willnot require any native implementation and can be reused for cordova/react native implementations. Individual SDK will may by default be implemented in pure JavaScript or as Cordova/React plugins. 
+
+## Advantages of the solution 1
+
+- Significantly reduced effort to develop and maintain Cordova Support by reusing available components.
+- Aligned with the way cordova works (single plugin to do specific *native* job, integrated with others in JavaScript)
+- Ability to reuse existing cordova plugins that perform native tasks (push notifications, sync, auth)
+- Ability to wrap existing JavaScript solutions and use them without 
+- Ability to select and use only elements that are needed in the app.
```

Comment:
I don't understand this, from a technical perspective. Ability to select what exactly?
Is this about loading/including of libraries? What's the alternative to this that means all elements are 'selected'?